### PR TITLE
Add a placeholder post for Hatsumi Taikai 2024

### PR DESCRIPTION
**Pull request by Iipin**

### CODE REVIEW

File: _posts/2023-10-06-hatsumi-taikai.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+* Data: 17-18 lutego 2024 (sobota-niedziela)
+* Miejsce: TBD, Kraków
+* Zasady: EMA Riichi 2016
+* Hanchany: 8
+* Wpisowe: TBD, ok. 150 PLN
+* Limit osób: 84
+
+* Obserwator EMA: TBD
+* Sędziowie: TBD
```
Comment (m4tx) [inline _posts/2023-10-06-hatsumi-taikai.markdown:21]: I suppose you can put me here

File: _posts/2023-10-06-hatsumi-taikai.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title:  "Hatsumi Taikai 2024"
```
Comment (m4tx) [inline _posts/2023-10-06-hatsumi-taikai.markdown:2]: I thought we agreed the name will not have the current year?

File: _posts/2023-10-06-hatsumi-taikai.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+categories: tournaments
+lang: pl
+---
+
+Zapraszamy Was na drugi turniej MERS w Krakowie!
+
+## O turnieju
+
+* Data: 17-18 lutego 2024 (sobota-niedziela)
+* Miejsce: TBD, Kraków
```
Comment (m4tx) [inline _posts/2023-10-06-hatsumi-taikai.markdown:14]: Let's put WMiI, it's not likely to change at this point

**Review by m4tx — CHANGES_REQUESTED** [review]

File: _posts/2023-10-06-hatsumi-taikai.markdown [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* Obserwator EMA: TBD
+* Sędziowie: TBD
+
+## Agenda
+
+TBD
+
+## Rejestracja
+
+TBD
```
Comment (m4tx) [inline _posts/2023-10-06-hatsumi-taikai.markdown:29]: Please, could we add registration link?

File: _posts/2023-10-06-hatsumi-taikai.markdown (hunk 2 above)
Comment (5zlotych) [reply _posts/2023-10-06-hatsumi-taikai.markdown:2]: > I thought we agreed the name will not have the current year?
we did

**Review by m4tx — CHANGES_REQUESTED** [review]

File: _posts/2023-10-06-hatsumi-taikai.markdown (hunk 4 above)
Comment (m4tx) [reply _posts/2023-10-06-hatsumi-taikai.markdown:29]: As discussed on Discord - let's wait until we confirm the place.

File: _posts/2023-10-06-hatsumi-taikai.markdown (hunk 3 above)
Comment (m4tx) [reply _posts/2023-10-06-hatsumi-taikai.markdown:14]: As discussed on Discord - let's wait until we confirm the place.

File: _posts/2023-10-06-hatsumi-taikai.markdown (hunk 4 above)
Comment (m4tx) [inline _posts/2023-10-06-hatsumi-taikai.markdown:29]:
```suggestion
TBD - [wyczekujcie ogłoszenia na naszym fanpage'u!](https://www.facebook.com/chombo.club)
```

**Review by m4tx — CHANGES_REQUESTED** [review]

**Review by m4tx — APPROVED** [review]

**Pull request merged**